### ISSUE TITLE
[cli] (starboard 0.7.1 get report) panic: runtime error: invalid memory address or nil pointer dereference

### ISSUE THREAD

**Issue by rurus9** [aquasecurity/starboard#292]
Clean install (starboard release 0.7.1 from https://github.com/aquasecurity/starboard/releases/download/v0.7.1/starboard_linux_x86_64.tar.gz):

```
$ starboard init
$ starboard scan vulnerabilityreports --namespace debian9 deployments/debian9
```
```
$ starboard get report --namespace debian9 deployments/debian9

<html>
        <head>
        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
                <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;700&display=swap" rel="stylesheet">
                <title>
        Starboard Security Report - debian9/Deployment/debian9
</title>
        </head>
        <body style="font-family: 'Lato', sans-serif;">

        <style>
  a {
    color: inherit;
  }
  @media print {
    .container {
        display: inline;
    }
  }
  </style>
  <div class="container border-right border-left" style="height: 100%; overflow: scroll;">
            <div class="col mt-5">
                <div class="row text-center">
                    <img style="width: 25vh" class="mx-auto" src="https://www.aquasec.com/wp-content/uploads/2016/05/aqua_logo_fullcolor.png" alt="">
                </div>
                <div class="row mt-4 text-center">
                    <h2 class="text-muted mx-auto">Starboard Security Report for<br>
                </div>
                <div class="row text-center">
                    <h2 class="text-muted mx-auto">Deployment debian9 in namespace debian9</h2>
                </div>

                  
                <div class="row mt-5 px-3">
                    <h4>Table Of Contents</h4>
                </div>
                <div class="row">
                    <ul>
                        
                        <li>
                            <a href="#vuln_header">Vulnerabilities</a></li>
                            <ul>
                              
                                <li><a href="#vulns_container_debian9">debian9</a></li>
                              
                            </ul>
                        </li>
                        
                        panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x178 pc=0x14528f1]

goroutine 1 [running]:
github.com/aquasecurity/starboard/pkg/report/templates.(*ReportPage).StreamBody(0xc000610940, 0xc000176230)
        pkg/report/templates/default.qtpl:106 +0x1b1
github.com/aquasecurity/starboard/pkg/report/templates.StreamPageTemplate(0xc000176230, 0x19c0ec0, 0xc000610940)
        pkg/report/templates/basicpage.qtpl:20 +0xae
github.com/aquasecurity/starboard/pkg/report/templates.WritePageTemplate(0x197eaa0, 0xc000010018, 0x19c0ec0, 0xc000610940)
        pkg/report/templates/basicpage.qtpl:23 +0x5c
github.com/aquasecurity/starboard/pkg/report.(*htmlReporter).GenerateReport(0xc000571da0, 0xc0000f9480, 0xa, 0x7fff9fa5d813, 0x7, 0x7fff9fa5d7ff, 0x7, 0x197eaa0, 0xc000010018, 0x7fff9fa5d813, ...)
        /home/runner/work/starboard/starboard/pkg/report/html.go:62 +0x310
github.com/aquasecurity/starboard/pkg/cmd.NewGetReportCmd.func1(0xc000170dc0, 0xc00023b440, 0x1, 0x3, 0x0, 0x0)
        /home/runner/work/starboard/starboard/pkg/cmd/get_report.go:54 +0x2d8
github.com/spf13/cobra.(*Command).execute(0xc000170dc0, 0xc00023b410, 0x3, 0x3, 0xc000170dc0, 0xc00023b410)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:842 +0x47c
github.com/spf13/cobra.(*Command).ExecuteC(0xc000377600, 0x5, 0x19a7fe0, 0x28)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:950 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:887
github.com/aquasecurity/starboard/pkg/cmd.Run(0x1960c34, 0x5, 0x19a7fe0, 0x28, 0x1967e10, 0x14, 0xc00003c1e0, 0x6, 0x6, 0x197eaa0, ...)
        /home/runner/work/starboard/starboard/pkg/cmd/root.go:62 +0xba
main.main()
        /home/runner/work/starboard/starboard/cmd/starboard/main.go:33 +0xda
```
starboard get vulnerabilities seems to work OK
```
$ starboard get vulnerabilities --namespace debian9 deployments/debian9
apiVersion: aquasecurity.github.io/v1alpha1
items:
- apiVersion: aquasecurity.github.io/v1alpha1
  kind: VulnerabilityReport
  metadata:
    creationTimestamp: "2020-12-30T10:58:00Z"
    generation: 1
    labels:
      starboard.container.name: debian9
      starboard.resource.kind: Deployment
      starboard.resource.name: debian9
      starboard.resource.namespace: debian9
    name: deployment-debian9-debian9
    namespace: debian9
    ownerReferences:
    - apiVersion: apps/v1
      kind: Deployment
      name: debian9
      uid: 103dac40-4761-429e-9f78-831d5c862665
    resourceVersion: "36157183"
    selfLink: /apis/aquasecurity.github.io/v1alpha1/namespaces/debian9/vulnerabilityreports/deployment-debian9-debian9
    uid: 56399794-587d-4343-b46e-5fb38eec25fd
  report:
    artifact:
      repository: arunvelsriram/utils
      tag: latest
    registry:
      server: index.docker.io
    scanner:
      name: Trivy
      vendor: Aqua Security
      version: 0.14.0
    summary:
      criticalCount: 0
      highCount: 3
      lowCount: 170
      mediumCount: 233
      noneCount: 0
      unknownCount: 0
    updateTimestamp: "2020-12-30T10:57:59Z"
    vulnerabilities:
    - fixedVersion: 1.6.12ubuntu0.2
      installedVersion: 1.6.12
      links: []
      primaryLink: https://avd.aquasec.com/nvd/cve-2020-27350
      resource: apt
      severity: MEDIUM
      title: ""
      vulnerabilityID: CVE-2020-27350
    - fixedVersion: 1.6.12ubuntu0.1
      installedVersion: 1.6.12
      links: []
      primaryLink: https://avd.aquasec.com/nvd/cve-2020-3810
      resource: apt
      severity: MEDIUM
      title: ""
      vulnerabilityID: CVE-2020-3810
        (...)
    - fixedVersion: 2:8.0.1453-1ubuntu1.4
      installedVersion: 2:8.0.1453-1ubuntu1.3
      links: []
      primaryLink: https://avd.aquasec.com/nvd/cve-2019-20807
      resource: xxd
      severity: LOW
      title: 'vim: users can execute arbitrary OS commands via scripting interfaces in the rvim restricted mode'
      vulnerabilityID: CVE-2019-20807
kind: VulnerabilityReportList
metadata: {}
```
**Environment:**

- Starboard version (use `starboard version`): {Version:0.7.1 Commit:3c4342d2f344dd3aeea3fb1083c9d7a499bbcf2f Date:2020-12-05T18:05:28Z}
- Kubernetes version (use `kubectl version`): Client Version: version.Info{Major:"1", Minor:"20", GitVersion:"v1.20.1", GitCommit:"c4d752765b3bbac2237bf87cf0b1c2e307844666", GitTreeState:"clean", BuildDate:"2020-12-18T12:09:25Z", GoVersion:"go1.15.5", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"16", GitVersion:"v1.16.13", GitCommit:"39a145ca3413079bcb9c80846488786fed5fe1cb", GitTreeState:"clean", BuildDate:"2020-07-15T16:10:14Z", GoVersion:"go1.13.9", Compiler:"gc", Platform:"linux/amd64"}

- OS (macOS 10.15, Windows 10, Ubuntu 19.10 etc): Debian bullseye/sid

**Comment by rurus9** [comment]
When I use starboard 0.6.0 to get report it works:
```
$ starboard-0.6.0 version
Starboard Version: {Version:0.6.0 Commit:ad6b9b99931d7524806f79b1329fdda82c75a715 Date:2020-10-14T20:14:11Z}

$ starboard-0.6.0 get report --namespace debian9 deployments/debian9

<html>
        <head>
        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
                <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;700&display=swap" rel="stylesheet">
                <title>
        Starboard Security Report - debian9/Deployment/debian9
</title>
        </head>
        <body style="font-family: 'Lato', sans-serif;">

        <style>
  a {
    color: inherit;
  }
  @media print {
    .container {
        display: inline;
    }
  }
  </style>
  <div class="container border-right border-left" style="height: 100%; overflow: scroll;">
            <div class="col mt-5">
                <div class="row text-center">
                    <img style="width: 25vh" class="mx-auto" src="https://www.aquasec.com/wp-content/uploads/2016/05/aqua_logo_fullcolor.png" alt="">
                </div>
                <div class="row mt-4 text-center">
                    <h2 class="text-muted mx-auto">Starboard Security Report for<br>
                </div>
                <div class="row text-center">
                    <h2 class="text-muted mx-auto">Deployment debian9 in namespace debian9</h2>
                </div>

                  
                <div class="row mt-5 px-3">
                    <h4>Table Of Contents</h4>
                </div>
                <div class="row">
                    <ul>
                        
                        <li>
                            <a href="#vuln_header">Vulnerabilities</a></li>
                            <ul>
                              
                                <li><a href="#vulns_container_debian9">debian9</a></li>
                              
                            </ul>
                        </li>
                        
                        
                    </ul>
                </div>


                
                <!-- Vulnerabilities -->
(...) 

                <!-- Config Audits -->
                
            </div>
        </div>

        </body>
</html>


```

**Comment by danielpacak** [comment]
Thank you reporting this issue @rurus9 I can reproduce it in my env and we already have PR to fix it #295
